### DISCUSSION TITLE
Error para actulizar estadisiticas de Company comunicathor, Solo muestra data hasta mayo 2023

### DISCUSSION THREAD

**Discussion by Helenlop** [community/community#127739] [Projects and Issues]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Error para actulizar estadisiticas de Company comunicathor, Solo muestra data hasta mayo 2023

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Abbas-Razaa** [comment]
Verifica la Fuente de Datos:

Asegúrate de que la fuente contiene datos más allá de mayo de 2023.
Revisa Scripts ETL:

Verifica que los scripts no tienen filtros que limiten los datos a mayo de 2023.
Actualiza Herramienta de Reportes:

Confirma que la herramienta de visualización esté actualizada y configurada para incluir datos recientes.
Configura Fechas Correctamente:

Asegúrate de que los parámetros de fecha en los reportes incluyan datos posteriores a mayo de 2023.
Revisa Logs y Errores:

Examina los logs del sistema y mensajes de error para identificar problemas.
Ejemplo SQL para verificar datos recientes:

> SELECT *
> FROM estadisticas
> WHERE fecha >= '2023-05-01'
> AND fecha <= CURRENT_DATE;

**Comment by queenofcorgis** [comment]
Hi @Helenlop , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
